tools/tests/organizations.js

```diff
@@ -36,54 +36,3 @@ selftest.define("organizations - logged out", function () {
   run.stop();
 
 });
-
-// For now, this test only runs from checkout with a universe file
-// pointing to a testing meteor-accounts server (e.g. one deployed with
-// Meteor.settings.testing = true).
-selftest.define("organizations", ["net", "slow", "checkout"], function () {
```

Comment:
Is this test useless without the notion of a "universe file" now, or just more trouble to fix than it's worth?